content/03.results.md

```diff
@@ -43,21 +43,21 @@ Pangenomes that contain genomes with the RefSeq exclusion criteria of "many fram
 ](images/frameshift_fig.png){#fig:frameshift_fig}
 
 We next investigated whether other pangenome metrics were well correlated between k-mer-based and gene-based methods using pangenomes that did not contain genomes excluded from RefSeq for having many frameshifted proteins.
-For these 13 pangenomes, the percent of k-mers or genes predicted to be part of the core, shell, or cloud genome was strongly correlated (**Figure @fig:pg_fig**).
+For these 13 pangenomes, the percent of k-mers or genes predicted to be part of the core, shell, or cloud pangenome was strongly correlated (**Figure @fig:pg_fig**).
 We also compared whether pangenomes would be designated as open or closed by calculating the alpha value for the Heaps law model. 
 Alpha values were strongly correlated between gene- and k-mer based pangenomes (**Figure @fig:pg_fig**). 
 
 ![
 **Pangenome metrics strongly correlate between gene- and k-mer-based pangenomes.** Pangenome categories core, shell, and cloud refer to genes or k-mers shared between the majority (>95%), some, or singleton genomes in the pangenome. Alpha is a value from Heaps law used to estimate whether a pangenome is open or closed.
 ](images/pg_fig.png){#fig:pg_fig height=2.5in}
 
-Taken together, these results show that reduced alphabet k-mers can accurately estimate pangenomes from bacterial and archaeal genomes.
+Taken together, these results show that reduced alphabet k-mers can accurately estimate key characteristics of pangenomes from bacterial and archaeal genomes.
 
 ## Jaccard containment between reduced alphabet k-mers and k-mers in databases accurately predicts open reading frames in short sequencing reads
 
-We next sought to determine whether open reading frames could be accurately predicted directly from short sequencing reads, as this would enable k-mer-based pangenome analysis from short reads.
+We next sought to determine whether open reading frames could be accurately predicted directly from short sequencing reads, as this would enable k-mer-based pangenome analysis without assembly.
```

Comment:
😆 i switched from this and back to it like 3 times before settling on "from short reads" -- will keep as "without assembly" now!